webui/tests/cypress/integration/02-navigation.spec.ts

```diff
@@ -49,6 +56,7 @@ describe('Navigation', () => {
 
   describe('side menu buttons', () => {
     beforeEach(() => {
+      cy.restoreLocalStorageCache([ STORAGE_KEY_AUTH ]);
       cy.visit('/det');
```

Comment:
**thought:** should we take this out or only apply it one time via `before()`? With the idea that all the subsequent navigation menu should work no matter what page you are on.